pkg/mac/common.sh

```diff
@@ -19,7 +19,10 @@
 
 set -e -x
 
-export ARCH=${ARCH:-arm64}
+if [ ! -n "$ARCH" ]; then
+   export ARCH=$(uname -m)
```

Comment:
Shall we normalize the output as:

```shell
$(uname -m | sed -r 's/aarch64/arm64/g' |  awk '!/arm64/{$0="amd64"}1')
```

?

Reply:
There is no need, this script is only used on Mac systems.